apps/ai-image-tagging/frontend/src/components/AITagView/AITagView.js

```diff
@@ -46,17 +47,24 @@ export class AITagView extends React.Component {
   }
 
   validateImage = async () => {
+    const FILE_LIMIT = 5000000; // 5MB limit from AWS Rekognition
```

Comment:
I'd call it `MAX_FILE_SIZE`. Also 5MB is `5 * Math.pow(2, 20)`

Reply:
I wasn't sure if they are strict on binary vs decimal, hence played it safe.